autoload/sneak.vim

```diff
@@ -189,6 +189,10 @@ func! sneak#to(op, input, inputlen, count, register, repeatmotion, reverse, incl
   let w:sneak_hl_id = matchadd('Sneak',
         \ (s.prefix).(s.match_pattern).(s.search).'\|'.curln_pattern.(s.search))
 
+  if a:inputlen > 1
```

Comment:
Could also check `0 != hlID('SneakCurrent')` here, but `matchadd()` seems to be ok with non-existent hl groups.